omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -132,7 +132,7 @@ private void BuildJSON() {
 			try {
 				initSwagger();
 				addPaths();
-				addDefaultDefinitions();
+				//addDefaultDefinitions();
```

Comment:
Is there a reason we don't want this? And if so, can we just delete the method?

Reply:
yes sure